sql/aiven_extras.sql

```diff
@@ -1,3 +1,12 @@
+-- Check that if schema already exist
```

Comment:
"Abort if schema already exists"?

Reply:
Thanks for the comment, please check the latest version. Need to check the schema's owner whether is an unprivileged user.